operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -767,6 +767,10 @@ def addNetworkAccessPoint(self, server, nap, is_server_template):
     nap_protocol = getNAPProtocol(nap, server, self.env.getDomain(), is_server_template)
 
     if istio_enabled == 'true':
+      name = nap.getName()
+      if name.startswith('http-') or name.startswith('tcp-') or name.startswith('tls-') or name.startswith('https-'):
```

Comment:
Could this be fragile? The customer could have a nap name that starts with one of these prefixes. Do we have some earlier validation that ensures that the only names with these prefixes would be naps that were edited by our Istio functionality?

Reply:
I could be wrong but I couldn't find any validation for naps that have these prefixes edited by our Istio functionality. @jshum2479 is there validation for these naps that I didn't see?

Reply:
This function is to write out the topology in the introspector output (not config override), it is trying to format the name in istio ways.  It's fine to leave it here if the name used by  customer already had that convention and skipping it.